docker/default.conf

```diff
@@ -0,0 +1,102 @@
+  rewrite ^/redhat/(.*) https://pkg.jenkins.io/redhat/$1 permanent;
+  rewrite ^/opensuse/(.*) https://pkg.jenkins.io/opensuse/$1 permanent;
+  rewrite ^/debian/(.*) https://pkg.jenkins.io/debian/$1 permanent;
+
+  # convenient short URLs
+  rewrite ^/issue/(.+)          https://issues.jenkins-ci.org/browse/JENKINS-$1 permanent;
+  rewrite ^/commit/core/(.+)    https://github.com/jenkinsci/jenkins/commit/$1 permanent;
+  rewrite ^/commit/(.+)/(.+)    https://github.com/jenkinsci/$1/commit/$2 permanent;
+  rewrite ^/pull/(.+)/([0-9]+)  https://github.com/jenkinsci/$1/pull/$2 permanent;
+
+  rewrite ^/maven-site/hudson-core /maven-site/jenkins-core permanent;
+
+  # https://issues.jenkins-ci.org/browse/INFRA-351
+  rewrite ^/maven-hpi-plugin(.*) http://jenkinsci.github.io/maven-hpi-plugin/$1 permanent;
+
+  # Probably not needed but, rating code moved a while ago
+  rewrite ^/rate/(.*) https://rating.jenkins.io/$1 permanent;
+  rewrite ^/census/(.*) https://census.jenkins.io/$1 permanent;
+  rewrite ^/jenkins-ci.org.key$ https://pkg.jenkins.io/redhat/jenkins.io.key permanent;
+
+  # permalinks
+  # - this one is referenced from 1.395.1 "sign post" release
+  rewrite ^/why$            https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=53608972 permanent;
+  # baked in the help file to create account on Oracle for JDK downloads
+  rewrite ^/oracleAccountSignup$    http://www.oracle.com/webapps/redirect/signon?nexturl=http://jenkins-ci.org/ permanent;
+  # to the donation page
+  rewrite ^/donate$        https://wiki.jenkins-ci.org/display/JENKINS/Donation permanent;
+  # CLA links used in the CLA forms
+  rewrite ^/license$        https://wiki.jenkins-ci.org/display/JENKINS/Governance+Document#GovernanceDocument-cla permanent;
+  rewrite ^/licenses$        https://wiki.jenkins-ci.org/display/JENKINS/Governance+Document#GovernanceDocument-cla permanent;
+  # used to advertise the project meeting
+  rewrite ^/meetings/$        https://wiki.jenkins-ci.org/display/JENKINS/Governance+Meeting+Agenda permanent;
+  # used from friends of Jenkins plugin to link to the thank you page
+  rewrite ^/friend$        https://wiki.jenkins-ci.org/display/JENKINS/Donation permanent;
+  # used by Gradle JPI plugin to include fragment
+  rewrite ^/gradle-jpi-plugin/latest$    https://raw.github.com/jenkinsci/gradle-jpi-plugin/master/install permanent;
+  # used when encouraging people to subscribe to security advisories
+  rewrite ^/advisories$        https://wiki.jenkins-ci.org/display/JENKINS/Security+Advisories permanent;
+  # used in slides and handouts to refer to survey
+  rewrite ^/survey$        http://s.zoomerang.com/s/JenkinsSurvey permanent;
+  # used by RekeySecretAdminMonitor in Jenkins
+  rewrite ^/rekey$            https://wiki.jenkins-ci.org/display/SECURITY/Re-keying permanent;
+  # persistent Google hangout link
+  rewrite ^/hangout$        https://plus.google.com/hangouts/_/event/cjh74ltrnc8a8r2e3dbqlfnie38 permanent;
```

Comment:
I'm impressed this link still works

Reply:
This will be exciting, having all these redirects managed in this repo and stuff